regolith/helpers/u_milestonehelper.py

```diff
@@ -16,8 +16,9 @@
 
 def subparser(subpi):
     subpi.add_argument("projectum_id", help="the id of the projectum")
-    subpi.add_argument("--number",
-                        help="number of the milestone to update from numbered list")
+    subpi.add_argument("--index",
+                        help="index of the item in the enumerated list chosen to update",
```

Comment:
let's remove the word "chosen" maybe? it doesn't read quite right to me....

Reply:
done!